test/conformance/platform/urInit.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: MIT
+#include <uur/checks.h>
+
+TEST(urInitTest, Success) {
+  ur_platform_init_flags_t platform_flags = 0;
+  ur_device_init_flags_t device_flags = 0;
+  ASSERT_SUCCESS(urInit(platform_flags, device_flags));
+
+  // TODO - Question? Should we call urTearDown() in these tests. We aren't
```

Comment:
IMO yes, we should always all resources allocated in a test.

Reply:
👍